src/cocoa_window.m

```diff
@@ -1511,6 +1511,16 @@ void _glfwPlatformSetCursorMode(_GLFWwindow* window, int mode)
     return _glfw.ns.keyName;
 }
 
+const short int _glfwPlatformGetKeyScancode(int key)
+{
+    if(key <= -1 || key >= (sizeof(_glfw.ns.nativeKeys) / sizeof(_glfw.ns.nativeKeys[0])))
+    {
```

Comment:
This test should be in `glfwGetKeyScancode` to avoid repetition, and it should emit `GLFW_INVALID_ENUM` if the key is invalid. See `glfwGetKey` for a similar test.